diff --git a/kernel/lsm_hooks.c b/kernel/lsm_hooks.c
index 62de1bdb..83efc925 100644
--- a/kernel/lsm_hooks.c
+++ b/kernel/lsm_hooks.c
@@ -84,7 +84,6 @@ static int ksu_inode_rename(struct inode *old_inode, struct dentry *old_dentry,
 	return 0;
 }
 
-#if defined(CONFIG_KSU_MANUAL_HOOK) && !defined(CONFIG_KSU_SUSFS)
 static int ksu_task_fix_setuid(struct cred *new, const struct cred *old,
 			       int flags)
 {
