--- a/init/Kconfig
+++ b/init/Kconfig
@@ -133,7 +133,7 @@
 
 config WERROR
 	bool "Compile the kernel with warnings as errors"
-	default y
+	default n
 	help
 	  A kernel build should not cause any compiler warnings, and this
 	  enables the '-Werror' flag to enforce that rule by default.
--- a/Makefile
+++ b/Makefile
@@ -809,7 +809,7 @@ stackp-flags-$(CONFIG_STACKPROTECTOR_STRONG)      := -fstack-protector-strong
 
 KBUILD_CFLAGS += $(stackp-flags-y)
 
-KBUILD_CFLAGS-$(CONFIG_WERROR) += -Werror
+KBUILD_CFLAGS-$(CONFIG_WERROR) += #-Werror
 KBUILD_CFLAGS += $(KBUILD_CFLAGS-y)
 
 ifdef CONFIG_CC_IS_CLANG
